actionlib_hatch_panel_params:
    elevator_timeout: 5 #in seconds
    pause_time_between_pistons: 0.1 #time to pause between piston actuations, in seconds

